docs/docs/20-usage/72-extensions/72-extensions.md

```diff
@@ -0,0 +1,34 @@
+# Extensions
+
+Woodpecker allows you to replace internal logic with external extensions by using pre-defined http endpoints.
+
+There are currently two types of extensions available:
```

Comment:
What is the second one? Is it possible to configure it globally like config extension?